### PR TITLE
Cross Signing redesign for composer 

### PR DESCRIPTION
**Pull request by t3chguy**
Fixes https://github.com/vector-im/riot-web/issues/11911
~~Missing tooltips, much larger task~~

image

Nad and I iterated on the sizing slightly as with any smaller than this it was hard to make out what it was

### CODE REVIEW

**Comment by t3chguy** [comment]
image

Tooltips being wrong orientation is a much larger issue which is outside the scope of this

File: src/components/structures/RoomView.js [hunk 1]
```diff
@@ -812,10 +812,10 @@ export default createReactClass({
         /* Check all verified user devices. */
         for (const userId of verified) {
             const devices = await cli.getStoredDevicesForUser(userId);
-            const allDevicesVerified = devices.every(({deviceId}) => {
-                return cli.checkDeviceTrust(userId, deviceId).isVerified();
+            const anyDeviceNotVerified = devices.some(({deviceId}) => {
+                return !cli.checkDeviceTrust(userId, deviceId).isVerified();
             });
-            if (!allDevicesVerified) {
+            if (anyDeviceNotVerified) {
```
Comment (dbkr) [inline src/components/structures/RoomView.js]: What's this change for? Isn't this a negation of the same predicate which is then be re-negated in the if clause?

Comment (t3chguy) [reply src/components/structures/RoomView.js]: Its just a bail-early optimization, the old code checked if every item was X, this code checks if any item is !X

Comment (dbkr) [reply src/components/structures/RoomView.js]: ah, of course

**Pull request merged**